tests/test_correlatednoise.py

```diff
@@ -36,14 +36,6 @@
 smallim_size = 16 # size of image when we test correlated noise properties using small inputs
 smallim_size_odd = 17 # odd-sized version of the above for odd/even relevant tests (e.g. draw)
 largeim_size = 12 * smallim_size # ditto, but when we need a larger image
-xlargeim_size =long(np.ceil(1.41421356 * largeim_size)) # sometimes, for precision tests, we 
-                                                        # need a very large image that will 
-                                                        # fit a large image within it, even if 
-                                                        # rotated
-
-# Decimals for comparison (one for fine detail, another for comparing stochastic quantities)
-decimal_approx = 2
-decimal_precise = 7
```

Comment:
Mostly (a) I don't like global variables in general, and (b) I don't really understand your use case of wanting to change the decimal value for lots of unit tests all at once.  Why did you find it useful to be able to do that?

Personally, I find it hard when reading a unit test to understand how precise each test really is when these variables are at module scope.  I can't see the definitions of these variables when I am looking at a test function, so these variables just obfuscate my understanding of how precise the test is.  The hard coded numbers in contrast are self-documenting.  

For instance, some of the times where you used `decimal_precise`, you had a comment saying that it was actually precise to machine precision.  In those cases, I switched to using decimal=12 and removed the comment.  That seems much clearer to me.  

In general, I feel like you should know how accurate a unit test is when you write it.  I generally start with decimal=5 or 6, and only drop it if I understand why the test should not be expected to pass at that level of precision.  Very often there is a good reason to only use decimal=2 or 3, but it is certainly test-specific, so I don't really see the point of having the decimal value for lots of tests set at module scope.  I'd rather have non-default decimal values be set for each test individually to the value appropriate for that specific test.